gemma/gemma-inl.h

```diff
@@ -848,32 +866,18 @@ void RangeChecks(size_t& max_tokens, size_t& max_generated_tokens,
 
 // Placeholder for internal test3, do not remove
 
-// Returns interleaved tokens: one from each query, followed by the second from
-// all queries, with EOS padding.
-static std::vector<int> InterleaveQueries(const MultiplePromptsTokens& queries,
-                                          const RuntimeConfig& runtime_config,
-                                          size_t& min_prompt_size,
-                                          size_t& max_prompt_size) {
+// Count the minimum/maximum size of prompts for interleave queries.
+static void InterleaveQueries(const MultiplePromptsTokens& queries,
```

Comment:
Please keep the std::vector return for now until we can remove the last reference to it.

Reply:
OK, let me drop that commit. :)